From 8cc583dde1c415a6e4452726431d76cc7f05c780 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Sat, 8 Oct 2022 09:43:39 +0800
Subject: [PATCH] Revert "Linux 5.10.147"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 014862eecf03f58066a957027dde73cbecdf4395.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 24110f834775..26a871eebe92 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 147
+SUBLEVEL = 146
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

